build/images/Dockerfile.build.controller.coverage

```diff
@@ -35,6 +35,10 @@ LABEL description="The Docker image to deploy the antrea-controller with code co
 
 USER root
 
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ca-certificates && apt-get clean && \
```

Comment:
Hi @tnqn not sure if this is necessary in OSS since we may have no public site access for now. Please double confirm. Thanks.

Reply:
I think it's unnecessary.

Reply:
I don't know if we need this, but if we do I would think the base image may be a better place to install this package

Reply:
Hi @antoninbas I think we already have this in base image, but the controller image is not longer based on the base image, so the file is missing after image split. Since @tnqn mentioned it's unnecessary, I can close it, and we may consider to add it when we do involve a new feature requires this.